src/commands/general/profile.ts

```diff
@@ -0,0 +1,29 @@
+import responses from "../../constants/responses";
+import type { Command } from "../../types/Command";
+import fetchUser from "../../util/fetchUser";
+import profileEmbed from "../../util/profileEmbed"; 
+export const command: Command = {
+  name: "profile",
+  description: "get the binarysearch profile of a user",
+  options: [
+    {
+    	name: 'query',
```

Comment:
could use a more descriptive arg name like user instead of query

Reply:
yeah, il change that